### ISSUE TITLE
Pagination: focus not removed after click

### ISSUE THREAD

**Issue by Meemaw** [Semantic-Org/Semantic-UI-React#2696]
### Steps

Click on any pagination number in docs. Item is bordered with blue (focused).

### Expected Result

It should not be focused after click.

### Version
0.79.0

**Comment by layershifter** [comment]
It should be focused, it's one of accessibility ideas. See also [Bootstrap](https://getbootstrap.com/docs/4.0/components/pagination/) docs.
